src/actions/guides/database/querying.cr

```diff
@@ -347,11 +347,22 @@ class Guides::Database::Querying < GuideAction
     When you have a model that is associated to another, your association is a method you can use
     to return those records.
 
+    ### Associations
+
+    Each association defined on your model will have a method that takes a block, and passed in the
+    query for that association.
```

Comment:
This is way cool. I didn't know we could do this.

Reply:
It's a good thing these docs are coming out!